src/lib/containers/widgets/facet-nav/FacetNav.tsx

```diff
@@ -122,18 +121,18 @@ const FacetNav: React.FunctionComponent<FacetNavProps> = ({
     return result
   }
 
-  const getShowMoreState = (): ShowMoreState => {
-    if (facetUiStateArray.length <= DEFAULT_VISIBLE_FACETS) {
-      return 'NONE'
-    }
+  const showMoreButtonState = useMemo<ShowMoreState>(() => {
     if (
       // if at least one item is hidden
       facetUiStateArray.find(item => item.isHidden === true)
     ) {
       return 'MORE'
+    } else if (facetUiStateArray.length <= DEFAULT_VISIBLE_FACETS) {
+      return 'NONE'
+    } else {
+      return 'LESS'
     }
-    return 'LESS'
-  }
+  }, [facetUiStateArray])
```

Comment:
Before, there was an edge case when there were fewer (or equal) total facets than the default visible.

1. Have, e.g. 1 plotted facet (default visible number of plots is 2)
2. Hide the facet plot
3. You would not see the "Show all plots" button, so the plot could not appear again w/o a refresh.

This fixes it by handling the case where a plot is hidden first. The behavior can be a little strange in some other cases, but not outright broken.

Reply:
Ok